docs/protocol-design/introduction.md

```diff
@@ -49,6 +49,37 @@ Although there is cross-over between the two main areas of the living whitepaper
+
+An alternative consensus protocol, Proof of Stake (PoS), was first introduced by Peercoin in 2012 [^8]. In a PoS system, participants vote with a weight equivalent to the amount of wealth they possess in a given cryptocurrency. With this arrangement, those who have a greater financial investment are given more power and are inherently incentivized to maintain the honesty of the system or risk losing their investment. PoS does away with the wasteful computation power competition, only requiring light-weight software running on low power hardware. While Nano uses a weighted-voting system ([ORV](/protocol-design#orv-consensus)) that can be compared to PoS, it differs from traditional PoS because there is not one monolithic blockchain that requires leader selection (i.e. a staker or a miner) to extend, representatives do not create or produce shared blocks (groups of transactions), each Nano account has its own blockchain that only the owner can modify (representatives can only modify their own blockchains), transactions are evaluated individually and asynchronously (a Nano block is a single transaction, not a group of transactions), anyone can be a representative, no funds are staked or locked up, users can remotely re-delegate their voting weight to anyone at any time, representatives do not earn transaction fees, and representatives cannot reverse transactions that nodes have locally confirmed (due to [block cementing](/glossary#cementing)).
+
+The original Nano (RaiBlocks) paper and first beta implementation were published in December, 2014, making it one of the first Directed Acyclic Graph (DAG) based cryptocurrencies [^9]. Soon after, other DAG cryptocurrencies began to develop, most notably DagCoin/Byteball and IOTA [^10], [^11]. These DAG-based cryptocurrencies broke the blockchain mold, improving system performance and security. Byteball achieves consensus by relying on a “main-chain” comprised of honest, reputable and user-trusted “witnesses”, while IOTA achieves consensus via the cumulative PoW of stacked transactions. Nano achieves consensus via a balance-weighted vote on conflicting transactions. This consensus system provides quicker, more deterministic transactions while still maintaining a strong, decentralized system. Nano continues this development and has positioned itself as one of the highest performing cryptocurrencies.
+
+## References
```

Comment:
No matter where the below references are on the page, this special footnote formatting forces them to the very bottom below a horizontal dotted line. I would suggest removing this this header as a result.

Reply:
Fixed!